### PR TITLE
olive-editor: Add version 0.1.0

### PR DESCRIPTION
**Pull request by omerakgoz34**
v0.1.0

### CODE REVIEW

File: bucket/olive-editor.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        "if (-not (Test-Path \"$dir\\shortcuts\")) { New-Item \"$dir\\shortcuts\" -ItemType File | Out-Null }",
+        "if (-not (Test-Path \"$dir\\portable\")) { New-Item \"$dir\\portable\" -ItemType File | Out-Null }"
+    ],
+    "persist": [
+        "config.xml",
+        "layout",
+        "shortcuts",
+        "portable"
+    ]
+}
```
Comment (Ash258) [inline bucket/olive-editor.json]:
```suggestion
{
    "version": "0.1.0",
    "description": "Non-linear video editor aiming to provide a fully-featured alternative to high-end professional video editing software.",
    "homepage": "https://olivevideoeditor.org",
    "license": "GPL-3.0-only",
    "architecture": {
        "64bit": {
            "url": "https://olivevideoeditor.org/dl/Olive-1e3cf53-Windows-x86_64.zip",
            "hash": "sha1:8d5cd30b71bc71178a3ccd857ec543002f7c0414"
        },
        "32bit": {
            "url": "https://olivevideoeditor.org/dl/Olive-1e3cf53-Windows-i686.zip",
            "hash": "sha1:99464160f6035ae8254b7a5a39c00a65ab476fb1"
        }
    },
    "extract_dir": "olive",
    "pre_install": [
        "'config.xml', 'layout', 'shortcuts' | ForEach-Object { New-Item \"$dir\\$_\" -ItemType File | Out-Null }",
        "New-Item \"$dir\\portable\" -ErrorAction SilentlyContinue | Out-Null"
    ],
    "bin": "olive-editor.exe",
    "shortcuts": [
        [
            "olive-editor.exe",
            "Olive Editor"
        ]
    ],
    "persist": [
        "config.xml",
        "layout",
        "shortcuts"
    ],
    "checkver": {
        "url": "https://github.com/olive-editor/olive/releases",
        "regex": "(?sm)Release build \\(([\\d.]+).*?Olive-(?<commit>\\w+)-Windows-"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://olivevideoeditor.org/dl/Olive-$matchCommit-Windows-x86_64.zip"
            },
            "32bit": {
                "url": "https://olivevideoeditor.org/dl/Olive-$matchCommit-Windows-i686.zip"
            }
        },
        "hash": {
            "url": "$url.sha1"
        }
    }
}

```

**Comment by omerakgoz34** [comment]
@Ash258 What is "package-fix-needed" label :?

**Comment by Ash258** [comment]
https://github.com/lukesampson/scoop-extras/pull/3540#discussion_r373771604

**Comment by Ash258** [comment]
Marked as resolved, but nothing was changed ...

You need to accept the suggestion or open the editor and replace your manifest with my changed.

A

**Comment by omerakgoz34** [comment]
> Marked as resolved, but nothing was changed ...
> 
> You need to accept the suggestion or open the editor and replace your manifest with my proposed changes.
> 
> A

I updated the file

**Pull request merged**